### PR TITLE
Display text on hover when it overflows

### PR DESCRIPTION
**Pull request by crisnicandrei**

### CODE REVIEW

File: src/app/file-browser/components/file-list-item/file-list-item.component.scss
Comment (meisekimiu) [inline src/app/file-browser/components/file-list-item/file-list-item.component.scss:128]: Is `article-name` the best name for this class? I feel like something like `item-name` might make more sense, though ultimately it doesn't matter since this classname is only used here.

**Review by meisekimiu — COMMENTED** [review]
This looks good, and the solution makes perfect sense! Since this is a style fix I would like to get this fix approved by the experience team before merging.

I'm noticing in Google Chrome that when hovering over text that fits in one line I still see a scrollbar appear next to the names, even if it doesn't do anything (this doesn't appear in Firefox though!). I do think we might want to fix that.

File: src/app/file-browser/components/file-list-item/file-list-item.component.scss
Comment (crisnicandrei) [reply src/app/file-browser/components/file-list-item/file-list-item.component.scss:128]: i think you are right!

**Comment by cecilia-donnelly** [comment]
@crisnicandrei fyi when @meisekimiu says this needs experience team approval we usually take screenshots or a video and share those in slack, tagging @k8lyn6 .

**Comment by crisnicandrei** [comment]
@meisekimiu I have pushed a new commit which hides the scrollbar but keeps the functionality. Let me know what you think!

**Review by meisekimiu — APPROVED** [review]
Thanks for fixing the scrollbar issue!

**Pull request merged**
